src/app/(dashboard)/criar-teste/_components/form/form.tsx

```
@@ -71,29 +93,33 @@ export default function TestForm() {
     groups: [],
   };
 
+  // Map UI question modes to API question modes
+  const mapQuestionMode = (
+    mode: string,
+  ): 'all' | 'unanswered' | 'incorrect' | 'bookmarked' => {
```

ESLint

[error] Move arrow function 'mapQuestionMode' to the outer scope.

(unicorn/consistent-function-scoping)